person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return false unless is_of_age? || @parent_permission
+
+    true
```

Comment:
- [ ] Good approach but I think it can be more simplified by a one-liner `is_of_age?() || @parent_permission`